autoload/vaffle/renderer.vim

```diff
@@ -3,9 +3,22 @@ set cpoptions&vim
 
 
 function! vaffle#renderer#render_item(item) abort
-  return printf('%s %s',
-        \ a:item.selected ? '*' : ' ',
-        \ a:item.basename . (a:item.is_dir ? '/' : ''))
+  if get(g:, 'vaffle_render_custom_icon', '') !=# ''
```

Comment:
I see. But I think it is better because If something goes wrong with your custom function, vaffle can provide a default rendering by including a try catch syntax.

What do you think? @cocopon

Reply:
It seems to be kind but if custom function has an error, Vaffle render default view and user cannot know why it doesn't work.

Reply:
Sure, you're right.